src/site/content/en/metrics/inp/index.md

```diff
@@ -44,20 +44,88 @@ This article explains how INP works, how to measure it, and offers advice for im
 
 ## What is INP?
 
-INP aims to represent a page's overall responsiveness by measuring all click, tap, and keyboard interactions made with a page. The longest of those observed interactions&mdash;with some exceptions noted below&mdash;is chosen as the page's INP value when the user is done with the page.
+INP aims to represent a page's overall responsiveness. It does so by measuring every click, tap, and keyboard interaction made with a page.
+
+### What is an interaction?
+
+An interaction is one or more discrete user input actions. More than one interaction may consist of what's considered a "logical user gesture". For a key board interaction, this would consist of both a `keydown` _and_ a `keyup` event.
+
+### What is an event?
+
+Interactions are driven by events. Events fire callbacks during the same logical user gesture. For example, "tap" interactions on a touchscreen device include multiple events, such as `pointerup`, `pointerdown`, and `click`. While JavaScript is a common driver of interactivity, interactions can also be driven by CSS and built-in browser controls (such as `<input>` elements, for example).
+
+The time it takes to queue up an interaction, including the time it takes for the event callbacks for the interaction to run, and the time it takes to present the next frame that shows the result of the interaction is known as the _event duration_.
+
+### How is INP calculated?
+
+Event duration for every interaction is recorded throughout the page lifecycle. In cases where there are few interactions made with a page&mdash;less than 50 to be exact&mdash;the interaction with the highest latency is chosen as the page's INP when the user is done with the page.
```

Comment:
Do we think we should recognise there is an INP per interaction? And a total page-level INP?

I note you're calling it "_event duration_" below, but it is the duration of the interaction until the next paint.

Or is that just confusing the situation and should we reserve INP for the metric?

Some more context from @mmocny here: https://twitter.com/mmocny/status/1578083571440537609?s=20&t=L_xMLmiuB804rpzLAU8RLw

```suggestion
### How is INP calculated for each event?

The time it takes to queue up an interaction, including the time it takes for the event callbacks for the interaction to run, and the time it takes to present the next frame that shows the result of the interaction is known as the _event duration_.

### How is INP calculated for the page?

Event duration for every interaction is recorded throughout the page lifecycle. In cases where there are few interactions made with a page&mdash;less than 50 to be exact&mdash;the interaction with the highest latency is chosen as the page's INP when the user is done with the page.

Reply:
I personally think this would add to confusion. INP—as I understand it—is a single value that's representative of overall page responsiveness by picking a single interaction from all observed interactions.

Reply:
Yeah I can see that. It's more obvious with CLS where there are individual "layout shifts" events and one "cumulative layout shift" metric. And for LCP it's the "LCP element" and the LCP metric.

For INP the full wording "Interaction to Next Paint" means it _could_ apply to both. But agree that would be confusing so maybe better to reserve this for the metric.

@philipwalton might be worth adding this to your INP terms doc, to state we use "INP" to refer to the metric, rather than the individual events?